vue-frontend/package.json

```diff
@@ -30,18 +29,15 @@
   "devDependencies": {
     "@babel/core": "^7.16.7",
     "@babel/eslint-parser": "^7.16.5",
-    "@vue/cli-plugin-babel": "~5.0.0-rc.2",
```

Comment:
Do not remove cli-plugin-babel and cli-plugin-babel. Prettier will not run without them